.utils/rust-by-example.js

```diff
@@ -0,0 +1,142 @@
+// This script expect Rust By Example to be in current directory
```

Comment:
Please add here where to get it from and how to use this file.

Reply:
Done!